src/components/Markdown/Markdown.style.ts

```diff
@@ -161,5 +161,7 @@ export const Caption = styled.figcaption`
 
 export const CenterEmbed = styled.div`
   display: flex;
+  flex-direction: column;
+  align-items: center;
```

Comment:
Added these as each embed link text is nested under `<CenterEmbed>` in `MarkdownLink.tsx`, which without these new properties will make the text sit to the right of the embed preview box.